tests/conftest.py

```diff
@@ -48,3 +57,41 @@ def recovery_codes(user) -> list[str]:
     ]
     StaticToken.objects.bulk_create(tokens)
     return [token.token for token in tokens]
+
+
+@pytest.fixture
+def mfa_admin_user(admin_user):
```

Comment:
Is `admin_user` a pytest-django specific fixture?

Reply:
yep!